Sources/TestingMacros/TestDeclarationMacro.swift

```diff
@@ -494,12 +494,13 @@ public struct TestDeclarationMacro: PeerMacro, Sendable {
 #if !SWT_NO_LEGACY_TEST_DISCOVERY
     // Emit a type that contains a reference to the test content record.
     let enumName = context.makeUniqueName(thunking: functionDecl, withPrefix: "__🟡$")
+    let unsafeKeyword: TokenSyntax? = isUnsafeKeywordSupported ? .keyword(.unsafe).with(\.trailingTrivia, .space) : nil
```

Comment:
```suggestion
    let unsafeKeyword: TokenSyntax? = isUnsafeKeywordSupported ? .keyword(.unsafe, trailingTrivia: .space) : nil
```

This should be more efficient, and these tokens are on a hot path. See https://github.com/swiftlang/swift-syntax/blob/6de157b65d57948c8904c3a326cbccfa261215ee/Sources/SwiftSyntax/generated/Tokens.swift#L230

Reply:
Sure thing.